llm_toolkit/prompt_builder.py

```diff
@@ -380,10 +380,14 @@ def __init__(self,
     self.base_template_file = self._find_template(template_dir, 'jvm_base.txt')
     self.data_filler_template_file = self._find_template(
         template_dir, 'jvm_specific_data_filler.txt')
-    self.problem_template_file = self._find_template(template_dir,
-                                                     'jvm_problem.txt')
     self.requirement_template_file = self._find_template(
         template_dir, 'jvm_requirement.txt')
+    self.problem_template_file = self._find_template(template_dir,
+                                                     'jvm_problem.txt')
+    self.constructor_template_file = self._find_template(
+        template_dir, 'jvm_problem_constructor.txt')
+    self.method_template_file = self._find_template(template_dir,
+                                                    'jvm_problem_method.txt')
```

Comment:
I can't find `jvm_problem_constructor.txt` and `jvm_problem_method.txt`, can you clarify?

Reply:
Sorry those are missed in the commit....